src/Xive/Props/Facets/EncodedProp.cs

```diff
@@ -0,0 +1,61 @@
+﻿//MIT License
+
+//Copyright (c) 2020 ICARUS Consulting GmbH
+
+//Permission is hereby granted, free of charge, to any person obtaining a copy
+//of this software and associated documentation files (the "Software"), to deal
+//in the Software without restriction, including without limitation the rights
+//to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//copies of the Software, and to permit persons to whom the Software is
+//furnished to do so, subject to the following conditions:
+
+//The above copyright notice and this permission notice shall be included in all
+//copies or substantial portions of the Software.
+
+//THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//SOFTWARE.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Yaapii.Atoms;
+
+namespace Xive.Props
+{
+    /// <summary>
+    /// Replaces ':' and ',' with '&#42' and '&#43'
+    /// </summary>
+    public sealed class EncodedProp : IScalar<string>
+    {
+        private readonly string origin;
+        private const string replaceColon = "&#42";
```

Comment:
I think we should use a more robust approach for encoding.
When you encode a comma to &#42, it might occur that someone sets a property value to "&#42" and this breaks the xive in another way.

The robust way would be to encode ALL special characters.

You can either build an own encoding, or use an existing one. An existing one could be 
https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlconvert.encodename?view=netcore-3.1

Please check if it encodes all characters.

Alternative:
WebUtility.UrlEncode()